stock_indicators/indicators/vwap.py

```diff
@@ -12,10 +12,11 @@
 @overload
 def get_vwap(quotes: Iterable[Quote], start: Optional[datetime] = None) -> "VWAPResults[VWAPResult]": ...
 @overload
-def get_vwap(quotes: Iterable[Quote], year: int,
+def get_vwap(quotes: Iterable[Quote], year: int, *,
              month: int = 1, day: int = 1,
              hour: int = 0, minute: int = 0) -> "VWAPResults[VWAPResult]": ...
-def get_vwap(quotes, start = None, month = 1, day = 1, hour = 0, minute = 0):
+def get_vwap(quotes: Iterable[Quote], start: Union[datetime, int, None] = None, *legacy_date_parts,
```

Comment:
Keyword argument before variable positional arguments list in the definition of get_vwap function